services/helper.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-plusplus */
+
+const sellerName = (id, sellers) => {
+  let name = '';
+  for (let i = 0; i < sellers.length; i++) {
+    if (sellers[i].id === id) {
+      name = sellers[i].name;
+      break;
+    }
+  }
+  return name;
+};
```

Comment:
Again, for this in similar functions, is there a reason you couldn't just return the name right away when it's found, and return an empty string if it gets through the entire loop? This could also reduce the if statement to 1 line.

Reply:
Please describe benefit in terms of time and/or space complexity or syntax. While I understand your approach not sure what I gain by above besides reducing few lines of code (which while a benefit, I don't consider gating).

Nevertheless I have made the change in interest of time and moving forward. Interested in understanding your concern still!

Reply:
I am simply reviewing the code. Aside from the obvious benefit of less lines, if the code is just as readable with fewer lines, why not change it while we can as this will be the code that merges into master.